static/js/constants.js

```diff
@@ -1,30 +1,90 @@
-export const COURSE_LIST_RESPONSE =
-  [
-    {
-      "id": 2,
-      "title": "Course xkGecQLCBCuI",
-      "enrollment_start": null,
-      "start_date": null,
-      "enrollment_url": null,
-      "prerequisites": null,
-      "program": 1
-    },
-    {
-      "id": 1,
-      "title": "Course zXHatukOIMTZ",
-      "enrollment_start": null,
-      "start_date": null,
-      "enrollment_url": null,
-      "prerequisites": null,
-      "program": 1
-    },
-    {
-      "id": 7,
-      "title": "Course plTYyFhMcEBj",
-      "enrollment_start": null,
-      "start_date": null,
-      "enrollment_url": null,
-      "prerequisites": null,
-      "program": 3
-    }
-  ];
+export const COURSE_LIST_RESPONSE = [
```

Comment:
Yes, we are also using this fake data in the API for now, since we don't have the real REST API available yet